From: Ben Hutchings <ben@decadent.org.uk>
Subject: alx: mark as staging
Date: Sun, 27 Jan 2013 23:43:01 +0000
Bug-Debian: http://bugs.debian.org/699129
Forwarded: not-needed

Since alx has not been accepted upstream (or even re-submitted for
many months), it should be treated as staging.

Also remove the netdev list and supported flag from its MAINTAINERS
entry.

---
Index: linux-3.2.46/MAINTAINERS
===================================================================
--- linux-3.2.46.orig/MAINTAINERS	2013-07-26 19:26:10.000000000 +0000
+++ linux-3.2.46/MAINTAINERS	2013-07-26 19:26:14.000000000 +0000
@@ -1326,10 +1326,8 @@
 M:	Stevent Li <steventl@qca.qualcomm.com>
 M:	Wu Ken <kenw@qca.qualcomm.com>
 M:	David Liu <dwliu@qca.qualcomm.com>
-L:	netdev@vger.kernel.org
 L:	nic-devel@qualcomm.com
 W:	http://wireless.kernel.org/en/users/Drivers/ethernet/alx
-S:	Supported
 F:	drivers/net/ethernet/atheros/alx/
 
 ATM
Index: linux-3.2.46/drivers/net/ethernet/atheros/alx/alx_main.c
===================================================================
--- linux-3.2.46.orig/drivers/net/ethernet/atheros/alx/alx_main.c	2013-07-26 19:26:13.000000000 +0000
+++ linux-3.2.46/drivers/net/ethernet/atheros/alx/alx_main.c	2013-07-26 19:26:14.000000000 +0000
@@ -41,6 +41,7 @@
 MODULE_AUTHOR("Qualcomm Corporation, <nic-devel@qualcomm.com>");
 MODULE_DESCRIPTION("Qualcomm Atheros Gigabit Ethernet Driver");
 MODULE_LICENSE("Dual BSD/GPL");
+MODULE_INFO(staging, "Y");
 
 static int alx_open_internal(struct alx_adapter *adpt, u32 ctrl);
 static void alx_stop_internal(struct alx_adapter *adpt, u32 ctrl);
